sechub-server/src/main/resources/db/migration/V28__enlarge_project_id.sql

```diff
@@ -0,0 +1,22 @@
+-- SPDX-License-Identifier: MIT
+
+-- Update project id : we now accept 765 (3 x 255) see ProjectIdValidation
```

Comment:
```suggestion
-- Update project id : we now accept 255 characters see ProjectIdValidation
```
There is no need to triple the value.
When the maximum is 255, then a `varchar(255)` value is sufficient.
(I have tested this successfully with Postgres and H2/Integrationtests)